test/invalidExamplesTests.ts

```diff
@@ -17,6 +17,6 @@ describe("Invalid examples", () => {
     if (result[0].details === undefined) {
       throw new Error("result[0].details === undefined")
     }
-    assert.notStrictEqual(result[0].details.url, undefined)
```

Comment:
What about keep the origin asset too?

Reply:
The original test was failing on this line, it's strange because I didn't modify any of related code and it's also failing on master branch. During debugging it seems like the data structure has changed and this test is no longer valid. Let me know what you think